modules/csharp/pages/_partials/pn-issues-list.adoc

```diff
@@ -1,5 +1,40 @@
 // BEGIN: tagged-inclusion -- issues-list for C#
 
+=== 2.8.6
+// tag::issues-2-8-6-net[]
+==== Deprecated
+// tag::deprecated-2-8-6-net[]
+// end::deprecated-2-8-6-net[]
+
+==== Enhancements
+// tag::enhancements-2-8-6-net[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=enhancements-2-8-6-net]
+
+
+// end::enhancements-2-8-6-net[]
+
+==== Fixed
+// tag::fixed-2-8-6-net[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=fixed-2-8-6-net]
+
+// * https://issues.couchbase.com/browse/CBL-1984[CBL-1984] -- https://issues.couchbase.com/browse/CBL-1984[Backport --  Support domain specific server authentication]
+// * https://issues.couchbase.com/browse/CBL-2001[CBL-2001] -- https://issues.couchbase.com/browse/CBL-2001[Backport --  Android v30 compatibility to 2.8.6]
+// * https://issues.couchbase.com/browse/CBL-2037[CBL-2037] -- https://issues.couchbase.com/browse/CBL-2037[Core pool size is too small]
```

Comment:
1984, 2001, and 2037 don't belong to .Net

Reply:
They were picked up because of android 2.8.6 version; just commented out in case they did apply. Will remove.

Reply:
Now staged here https://ibsoln.github.io/stage/stage280/couchbase-lite/current/csharp/release-notes.html